scarlet_extensions/Run/runner.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import scarlet.display
+from scarlet.initialization import build_initialization_coadd
+from ..initialization.detection import makeCatalog
+
+
+class Runner:
+    """ Class that lets scarlet run on a set of `Data` objects
+
+        This class makes running on batches of images easier.
+        It runs on multiple or single observations alike.
+
+        parameters
+        ----------
+        datas: list of Data objects
+            the list of Data objects to run scarlet on
```

Comment:
You might want to make it clear what the expected format of the data object is, eg. that it's a compressed numpy file with keys (`images` = 3D image cube, etc).